services/user/Invite/plugin/src/lib.rs

```diff
@@ -194,9 +201,9 @@ impl Inviter for InvitePlugin {
     fn generate_invite() -> Result<String, CommonTypes::Error> {
         let keypair = keyvault::generate_unmanaged_keypair()?;
 
-        let seed = rand::rand_bytes(8);
+        let seed = rand_bytes(8);
         let encrypted_private_key =
-            aes::with_password::encrypt(&seed, keypair.private_key.as_bytes());
+            aes::with_password::encrypt(&seed, keypair.private_key.as_bytes(), &keypair.public_key);
```

Comment:
Is `keypair.public_key` PEM? I'm somewhat concerned that this requires the PEM encoding created by rust to be identical to the encoding created by Botan. DER is more reliable when you need a canonical encoding.

Reply:
It is PEM. I can switch to der

Reply:
Done